Classes/Controller/SingleCollectionController.php

```diff
@@ -26,71 +26,71 @@
 use TYPO3\CMS\Core\Utility\GeneralUtility;
 
 use Kitodo\Dlf\Common\Document;
-use Slub\SlubDigitalcollections\Domain\Repository\KitodoDocumentRepository;
-use Slub\SlubDigitalcollections\Domain\Repository\KitodoStructuresRepository;
-use Slub\SlubDigitalcollections\Domain\Repository\KitodoCollectionsRepository;
-use Slub\SlubDigitalcollections\Domain\Repository\KitodoMetadataRepository;
+use Kitodo\Dlf\Domain\Repository\DocumentRepository as DocumentRepository;
```

Comment:
This should be sufficiant: 

```suggestion
use Kitodo\Dlf\Domain\Repository\DocumentRepository;
```